### ISSUE TITLE
FCOS模型运行tools/eval.py 报错

### ISSUE THREAD

**Issue by hujinda** [PaddlePaddle/PaddleDetection#716]
```
Error: ShapeError: broadcast dimension mismatch. Operands could not be broadcast together with the shape of X = [1760] and the shape of Y = [8, 17
60]. Received [1760] in X is not equal to [8] in Y at (/paddle/paddle/fluid/operators/elementwise/elementwise_op_function.h:145)
  [operator < elementwise_add > error]
terminate called without an active exception
```

用的是官网提供的模型fcos_r50_fpn_1x.pdparams，eval时出现错误，希望帮忙解答。

**Comment by qingqing01** [comment]
@hujinda  Thank! need to revert code in https://github.com/PaddlePaddle/PaddleDetection/pull/666/files#diff-37d7b652aa324ddc814384e18052aaa8 . We will fix it soon.

**Comment by willthefrog** [comment]
fixed in #718

**Comment by hujinda** [comment]
> fixed in #718

已解决，非常感谢，辛苦啦。
